### PR TITLE
 [AIRFLOW-5205] Check xml files with xmllint + Licenses

### PR DESCRIPTION
**Pull request by potiuk**
Make sure you have checked _all_ steps below.

### Jira

- [x] My PR addresses the following [Airflow Jira](https://issues.apache.org/jira/browse/AIRFLOW/) issues and references them in the PR title. For example, "\[AIRFLOW-XXX\] My Airflow PR"
  - https://issues.apache.org/jira/browse/AIRFLOW-5205

### Description

- [x] Here are some details about my PR, including screenshots of any UI changes:

### Tests

- [x] My PR adds the following unit tests __OR__ does not need testing for this extremely good reason:

### Commits

- [x] My commits all reference Jira issues in their subject lines, and I have squashed multiple commits if they address the same issue. In addition, my commits follow the guidelines from "[How to write a good git commit message](http://chris.beams.io/posts/git-commit/)":
  1. Subject is separated from body by a blank line
  1. Subject is limited to 50 characters (not including Jira issue reference)
  1. Subject does not end with a period
  1. Subject uses the imperative mood ("add", not "adding")
  1. Body wraps at 72 characters
  1. Body explains "what" and "why", not "how"

### Documentation

- [x] In case of new functionality, my PR adds documentation that describes how to use it.
  - All the public functions and the classes in the PR contain docstrings that explain what it does
  - If you implement backwards incompatible changes, please leave a note in the [Updating.md](https://github.com/apache/airflow/blob/master/UPDATING.md) so we can assign it to a appropriate release

### Code Quality

- [x] Passes `flake8`

### CODE REVIEW

File: airflow/_vendor/slugify/slugify.py [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
+"""Slugify !"""
```
Comment (ashb) [inline airflow/_vendor/slugify/slugify.py]: I'd rather we don't touch the files under airflow/_vendor/ unless we _have_ to.

Comment (ashb) [reply airflow/_vendor/slugify/slugify.py]: (Oh this change is probably from one of the other PRs isn't it? So that comment applies somewhere else)

**Comment by Fokko** [comment]
What's the point of opening this PR? It is impossible to review with some many dependencies on other PRs.

**Comment by feluelle** [comment]
Agree with Fokko. I personally don't want to be responsible for reviewing such a huge PR and maybe miss something - but I really like reviewing PR's in general :(

@potiuk do you think it is possible to split it into smaller ones?

**Comment by Fokko** [comment]
I've noticed that the PR's in the list are sequential, it starts with this one: https://github.com/apache/airflow/pull/5777

**Comment by potiuk** [comment]
@fokko @feluelle : apologies for that. i split those changes into smaller, fairly independent commits (hence the "depends on" in the PR title). And it was much easier to do the "series" of commits. They all depend on the #5777 and once I merge that one (Thanks @fokko for the approaval), I will actually change those commits/PR and make them totally separate (and not depending on each other).

File: airflow/_vendor/slugify/slugify.py (hunk 1 above)
Comment (potiuk) [reply airflow/_vendor/slugify/slugify.py]: Yeah. I will split those and exclude vendor from the original change. I thought I did that everywhere but I might have corrected vendor accidentally.

Comment (potiuk) [reply airflow/_vendor/slugify/slugify.py]: Removed in the first commit.

**Comment by potiuk** [comment]
Made the PR standalone (not depending on series of PRs

**Comment by potiuk** [comment]
This one is really small and mostly about consistency in licence. Rather straightforward (the only potential problem is necessity of xmllint  - but this is really not a problem because pre-commit will skip that check if you don't modify any of the xml files - so you won't need xmllint in most of the cases.

**Review by milton0825 — APPROVED** [review]

**Pull request merged**